src/style.css

```diff
@@ -0,0 +1,102 @@
+
+@font-face {
```

Comment:
Don't leave a redundant blank lines
```suggestion
@font-face {
```

Reply:
Redundant line has been removed